sources/packages/backend/libs/sims-db/src/entities/student-disability-status.type.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Disability status of student.
+ */
+export enum DisabilityStatus {
+  NotRequested = "Not requested",
+  Requested = "Requested",
+  PD = "PD",
+  PPD = "PPD",
```

Comment:
IMO, maybe adding a brief comments to each status maybe help the future developers